jerry-core/ecma/base/ecma-helpers-conversion.c

```diff
@@ -280,6 +280,11 @@ static const uint8_t ecma_uint4_clz[] = { 4, 3, 2, 2, 1, 1, 1, 1, 0, 0, 0, 0, 0,
  */
 #define EPSILON 0.0000001
 
+/**
+ * it's a constant that check ecma_utf8_string_to_number_by_radix radix arguments
```

Comment:
Comment: `Underscore is ignored when this option is passed.`

I know the names above does not follow the syntax (they are very old), but please rename
ALLOW_UNDERSCORE -> ECMA_CONVERSION_ALLOW_UNDERSCORE